ftr: Improvements in draft page

meetbhalodi11 opened the pull request:
1, UI improvements in drafts title, description and editor 
2, Edit mode button on top left UI changes
3, Editor info popup UI enhancements
Closes #12

Commit:
ftr: Improvements in draft page
UI improvements in drafts title, descritpion and editor
Edit mode button UI changes
Editor infor popup UI enhancements

vishal-kumar3 commented:
> 1, UI improvements in drafts title, description and editor 2, Edit mode button on top left UI changes 3, Editor info popup UI enhancements Closes #12

share the screenshots for the pages you changed.

vishal-kumar3 commented:
@meetbhalodi11, the editor info popup should only appear once for a draft and from then on only if user clicks the info button then only it should appear. You haven't fixed this yet.

meetbhalodi11 commented:
Can you elaborate on this behaviour?
Edit: From what I understood from this is if draft opens first time the only we want to show the dialog immediately. from second time onwards, it is not necessary.

Commit:
ftr: Improvements in draft page
the information dialog will be shown only once for every draft (instead of showing it everytime when draft opens)
